doc/supportedPlatformVersions.md

```diff
@@ -2,6 +2,52 @@
 
 ## dotnet
 
+### bookworm
+
+- 6.0.100-preview.1.21103.13
```

Comment:
Should we only support .NET 8 on bookworm? If we're progressively moving to a Stack based approach where each base OS has a specific set of languages/versions it supports, then bookworm might be a good place to start.

Reply:
Agreed. Updated.